src/compiler/checker.ts

```diff
@@ -180,7 +180,7 @@ namespace ts {
         IsForSignatureHelp = 1 << 4,    // Call resolution for purposes of signature help
     }
 
-    const enum AccessFlags {
+    export const enum AccessFlags {

```

Comment:
👀

Reply:
I moved it to types.ts in one iteration and forgot to un-export it when I moved it back